ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/byo-iam-role/cucushift-installer-rehearse-aws-ipi-byo-iam-role-workflow.yaml

```diff
@@ -1,9 +1,11 @@
 workflow:
   as: cucushift-installer-rehearse-aws-ipi-byo-iam-role
-  steps: 
+  steps:
     pre:
-    - chain: cucushift-installer-rehearse-aws-ipi-byo-iam-role-provision
+      - chain: cucushift-installer-rehearse-aws-ipi-byo-iam-role-provision
+      - ref: cucushift-installer-reportportal-marker
     post:
-    - chain: cucushift-installer-rehearse-aws-ipi-byo-iam-role-deprovision
+      - chain: cucushift-installer-rehearse-aws-ipi-byo-iam-role-deprovision
+      - ref: junit2reportportal
```

Comment:
I think the procedure should be,

1. Merge this PR ( without the append of step `junit2reportportal`, because the step does not contain logic to process `cucushift-installer-reportportal-marker` yet).
2. Merge https://github.com/openshift/release/pull/67092 ( with changes to contains logic to process `cucushift-installer-reportportal-marker`)
3. Use another PR to amend step `junit2reportportal` to workflows, and remove the step from the chain.

Reply:
we can merge https://github.com/openshift/release/pull/67092 firstly, then follow up this PR.

Reply:
https://github.com/openshift/release/pull/67092 is ready for review.